dapp-oeth/src/hooks/useSwapEstimator.js

```diff
@@ -374,6 +370,21 @@ const useSwapEstimator = ({
       }
     }
 
+    let amountReceived = amount
+
+    if (coinToSwap === 'sfrxeth') {
+      const [estimatedDeposit, frxEthMintPrice] = await Promise.all([
```

Comment:
Should we add some error handling?

Reply:
I think by default if this errors in the promise it would already throw an `unexpected_error`, do we want to do something different?

Reply:
Not needed in that case